radix-engine/src/blueprints/consensus_manager/validator.rs

```diff
@@ -155,6 +155,8 @@ impl NonFungibleData for UnstakeData {
 #[derive(Debug, Clone, PartialEq, Eq, ScryptoSbor)]
 pub enum ValidatorError {
     InvalidClaimResource,
+    InvalidGetRedemptionAmount,
+    UnexpectedDecimalComputationError,
```

Comment:
It would be great to have a test for this error.

Reply:
I believe with the current constraints on the resource system that this error would be impossible to get to (hence the "Unexpected" part of the error name)